biz.aQute.bndlib/src/aQute/bnd/osgi/Builder.java

```diff
@@ -1458,6 +1458,10 @@ public boolean doNotCopy(String v) {
 		return getDoNotCopy().matcher(v).matches();
 	}
 
+	public boolean doNotCopy(File from) {
+		return doNotCopy(from.getName()) || doNotCopy(from.getAbsolutePath());
```

Comment:
You will need to do a lot more work to match full paths on Windows. I suspect this is why the windows build failed.